mitosheet/src/mito/pro/taskpanes/ConditionalFormatting/ConditionalFormattingCard.tsx

```diff
@@ -145,7 +145,11 @@ const ConditionalFormattingCard = (props: ConditionalFormattingProps): JSX.Eleme
             {invalidColumnIDMessage}
             <Filter
                 filter={props.conditionalFormat.filters[0]}
-                columnDtype={undefined}
+                columnDtype={
+                    props.conditionalFormat.columnIDs.length > 0
+                    ? props.sheetData.columnDtypeMap[props.conditionalFormat.columnIDs[0]]
+                    : undefined
+                }
```

Comment:
Hm. Only showing specific filter conditions if there is one selected column ID feels more confusing than what we currently have. Too much of "what, where did they go"

I'm down to just keep all filter conditions for now, as the alternative feels like a bigger refactor than it's worth.

Reply:
Ok!